scrapi/harvesters/dryad.py

```diff
@@ -14,5 +19,17 @@ class DryadHarvester(OAIHarvester):
     url = 'http://www.datadryad.org/oai/request'
 
     base_url = 'http://www.datadryad.org/oai/request'
-    property_list = ['rights', 'format', 'relation', 'date', 'identifier', 'type']
+    property_list = ['rights', 'format', 'relation', 'date',
+                     'identifier', 'type', 'setSpec']
     timezone_granularity = True
+
+    def normalize(self, raw_doc):
+        str_result = raw_doc.get('doc')
+        result = etree.XML(str_result)
```

Comment:
If ```raw_doc.get('doc')``` is None, this will still throw an error, and 'doc' is required, so you might as well just do ```result = etree.XML(raw_doc['doc'])```